core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -150,17 +150,17 @@ public void seekTo(VideoPosition position) throws IllegalStateException {
     @Override
     public void stop() {
         reset();
+        listenersHolder.getStateChangedListeners().onVideoStopped();
     }
 
     @Override
     public void release() {
-        reset();
+        stop();
```

Comment:
are we positive we'll always want to do `stop()` when releasing, even if in the future our stop routine changes? or is it just mere coincidence that right now we do the same stuff? I would be cautious when calling a public method inside another. If you are already considering this then 👍